soc/arm/nuvoton_npcx/npcx9/soc.c

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2024 Nuvoton Technology Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/device.h>
+#include <zephyr/init.h>
+#include <soc.h>
+#include <zephyr/logging/log.h>
+
+LOG_MODULE_REGISTER(soc, CONFIG_SOC_LOG_LEVEL);
+
+static int soc_npcx9_init(void)
+{
+#ifdef CONFIG_NPCX_VCC1_RST_HANG_WORKAROUND
```

Comment:
Non-blocking, can you use `if (IS_ENABLED)` here?

Reply:
Done.